### PR TITLE
Add scroll depth tracking to 3 dvla page 

### PR DESCRIPTION
**Pull request by zilnhoj**
and remove scroll depth tracking from /tax-disc, aaib-reports and /business-finance-support-finder/search

### CODE REVIEW

File: app/assets/javascripts/analytics/scroll-tracker.js [hunk 1]
```diff
@@ -32,20 +32,17 @@
       ['Heading', 'Log in problems'],
       ['Heading', 'Manage your student finance']
     ],
-    '/tax-disc': [
-      ['Heading', 'Other ways to apply'],
-      ['Heading', 'Help with tax discs']
+    '/contact-the-dvla/y/driving-licences-and-applications': [
```
Comment (erik-eide) [inline app/assets/javascripts/analytics/scroll-tracker.js]: This first heading seems to be 'Driver licensing enquiries'

File: app/assets/javascripts/analytics/scroll-tracker.js [hunk 2]
```diff
@@ -32,20 +32,17 @@
...
+      ['Heading', 'Vehicle registration enquiries'],
+      ['Heading', 'When to contact DVLA']
     ],
-    '/aaib-reports': [
-      ['Percent', 25],
-      ['Percent', 50],
-      ['Percent', 75]
+    '/contact-the-dvla/y/vehicle-tax-and-sorn': [
```
Comment (erik-eide) [inline app/assets/javascripts/analytics/scroll-tracker.js]: And this heading is probably: 'Vehicle tax enquiries'

**Comment by erik-eide** [comment]
Other than two headings that might need updating, looks good to me

**Comment by erkede** [comment]
I've just rebased this with master, and forced pushed

**Comment by fofr** [comment]
Can this be closed?

**Comment by zilnhoj** [comment]
Rebased to reflect changes to heading names - ready for merge

**Pull request merged**